app/views/posts/_post.html.erb

```diff
@@ -1,3 +1,4 @@
+<% if current_user.friend?(post.user) || post.user.friend?(current_user) || current_user.id  == post.user.id %>
```

Comment:
- [ ] Instead of having the if/end statement create a helper method that has the if and renders a partial that contains the markup within the if/end statement instead of having the logic in the view